olp-cpp-sdk-dataservice-read/tests/CatalogRepositoryTest.cpp

```diff
@@ -183,9 +183,9 @@ TEST_F(CatalogRepositoryTest, GetLatestVersionOnlineOnlyFoundAndCacheWritten) {
         return boost::any{};
```

Comment:
Test name is then wrong, it says -> GetLatestVersionOnlineOnlyFoundAndCacheWritten
But you are not testing that the cache is written, you actually test that the cache is not written.